mk/top_of_makefile.mk

```diff
@@ -27,10 +27,12 @@ TARGET_VENDOR = $(word 2,$(TARGET_WORDS))
 TARGET_SYS = $(word 3,$(TARGET_WORDS))
 TARGET_ABI = $(word 4,$(TARGET_WORDS))
 
-# Cargo doesn't pass the ABI as part of TARGET on Mac OS X.
+# Cargo doesn't pass the ABI as part of TARGET on Mac OS X or Android.
```

Comment:
Yeah, you're right and it's not exactly cargo, but how the rust team named the platforms it builds for. I'm working on the wording to explain this.